tokio-postgres/src/lib.rs

```diff
@@ -103,7 +103,7 @@
 //!
 //! | Feature | Description | Extra dependencies | Default |
 //! | ------- | ----------- | ------------------ | ------- |
-//! | `runtime` | Enable convenience API for the connection process based on the `tokio` crate. | [tokio](https://crates.io/crates/tokio) 0.2 with the features `dns`, `net` and `time` | yes |
+//! | `runtime` | Enable convenience API for the connection process based on the `tokio` crate. | [tokio](https://crates.io/crates/tokio) 0.3 with the `io-util` feature | yes |
```

Comment:
The `io-util` feature is always enabled. `runtime` turns on `net` and `time`: https://github.com/sfackler/rust-postgres/blob/master/tokio-postgres/Cargo.toml#L28

Reply:
Oh, I had seen it but then lost it from my eyes when edited file :man_facepalming: 
`dns` feature still should be removed in the docs?

Reply:
Yeah, the dns feature doesn't exist anymore.